src/blockrelay/graphene.cpp

```diff
@@ -109,12 +105,15 @@ bool CGrapheneBlockTx::HandleMessage(CDataStream &vRecv, CNode *pfrom)
     CGrapheneBlockTx grapheneBlockTx;
     vRecv >> grapheneBlockTx;
 
+    auto pblock = thinrelay.GetBlockToReconstruct(pfrom);
+    std::shared_ptr<CGrapheneBlock> grapheneBlock = pblock->grapheneblock;
```

Comment:
make sure pblock->grapheneblock != nullptr since SetNull/ClearAllBlockData can set it to null.

Reply:
adding a DbgAssert here.